roles/openshift_service_catalog/files/kubesystem_roles_bindings.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kube-system-service-catalog
+objects:
+
+# this isn't being created in the correct namespace....
+- apiVersion: v1
+  kind: Role
+  metadata:
+    name: extension-apiserver-authentication-reader
+    namespace: kube-system
```

Comment:
sorry, that comment about it not being created in the right namespace was leftover before i moved it to its current location... the oc_process module should specify the module this is intended to be created in. If we don't need to specify a namespace in the metadata then I can remove this all together most likely (will double check)